bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/model/Rhythm.java

```diff
@@ -29,55 +31,55 @@
     private RhythmPos rhythmPos;
 
     public Boolean getRhythmConnected() {
-        return rhythmConnected;
+        return rhythmConnected == null ? Boolean.FALSE : rhythmConnected;
```

Comment:
Why not make `rhythmConnected` a primitive `boolean`?

Reply:
Fixed